components/PageButtons/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,125 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import "../index.css";
+import "../stories.js";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS Components/PageButtons"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
```

Comment:
I am missing the part:

`<Description markdown={readme} />`

`## Design Tokens`

`<DesignTokenDocBlock categoryName="PageButtons" viewType="table" />`

`## Stories`

`### Default`